src/openforms/js/components/admin/form_design/variables/VariablesEditor.js

```diff
@@ -51,14 +51,15 @@ const VariablesEditor = ({variables, onAdd, onChange, onDelete}) => {
           </TabList>
 
           <TabPanel>
-            <VariablesTable variables={componentVariables} />
+            <VariablesTable variables={componentVariables} onFieldChange={onFieldChange} />
```

Comment:
I haven't added all the missing props as this panel is for component variables which are readonly. So only the `onFieldChange` prop is relevant here (used to edit the registration options, the already existing `onChange` dispatches `CHANGE_USER_DEFINED_VARIABLE` so I can't use it :/)